### PR TITLE
Add possibility to associate Accounts with Organizations

### PR DESCRIPTION
**Pull request by joaodamasceno2001**

### CODE REVIEW

**Comment by joaodamasceno2001** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 1]
```diff
@@ -476,6 +486,7 @@ public void initialize(long groupId) throws InitializationException {
 			_invoke(() -> _addOrUpdateExpandoColumns(serviceContext));
 			_invoke(() -> _addOrUpdateKnowledgeBaseArticles(serviceContext));
 			_invoke(() -> _addOrUpdateOrganizations(serviceContext));
+			_invoke(() -> _addOrganizationsOnAccounts(serviceContext));
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:489]: add blank line before calling the method _addOrganizationsOnAccounts

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 2]
```diff
@@ -1304,6 +1315,62 @@ private Map<String, String> _addObjectDefinitions(
 		).build();
 	}
 
+	private void _addOrganizationsOnAccounts(ServiceContext serviceContext)
+		throws Exception {
+
+		String json = SiteInitializerUtil.read(
+			"/site-initializer/organizations-assignments.json",
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:1322]: rename json to accounts-organizations.json

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java [hunk 3]
```diff
@@ -1304,6 +1315,62 @@ private Map<String, String> _addObjectDefinitions(
 		).build();
 	}
 
+	private void _addOrganizationsOnAccounts(ServiceContext serviceContext)
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender/src/main/java/com/liferay/site/initializer/extender/internal/BundleSiteInitializer.java:1318]:
```suggestion
	private void _addAccountsOrganizations(ServiceContext serviceContext)
```

File: modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/organizations-assignments.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+[
+	{
+		"accountExternalReferenceCode": "TESTACCOUNT1",
+		"organizationExternalReferenceCode": [
```
Comment (nNilton) [inline modules/apps/site-initializer/site-initializer-extender/site-initializer-extender-test-bundle-1/src/main/resources/site-initializer/organizations-assignments.json:4]:
```suggestion
		"organizations": [
```

**Review by nNilton — CHANGES_REQUESTED** [review]

**Comment by joaodamasceno2001** [comment]
ci:test:sf

**Comment by joaodamasceno2001** [comment]
ci:forward

**Pull request closed without merging**